src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -0,0 +1,118 @@
+import { components } from 'src/api/types/generated-schema';
+
+import { Response } from './HttpService';
+
+export type ProjectSpecies = {
+  deliverableId: number;
```

Comment:
No deliverable ID will be needed on this entity. Basically, the species deliverables just exist, and implicitly "point to" the entire list of participant project species that exist at that moment. There is no direct link from a deliverable to a participant project species.

Reply:
Ok interesting, but makes sense. Will remove.